Cargo.toml

```diff
@@ -21,9 +21,12 @@ revm = { git = "https://github.com/risechain/revm", rev = "f473fb17f127a8ead2bcc
 # RPC Storage dependencies
 # TODO: Put these behind an RPC flag to not pollute the core
 # library with RPC network & transport dependencies, etc.
+alloy-consensus = { git = "https://github.com/alloy-rs/alloy", rev = "64feb9bc51c8021ea08535694c44de84222f474e" }
```

Comment:
Let's move core types to where `alloy-primitives` is above. This section is for RPC Storage dependencies that we'll make optional in the future.

Reply:
> are these just dev-dependencies at the moment?

yes, all the added deps are for `tests/`

Reply:
Then please move them to `[dev-dependencies]`.